handlers/Careers/components/CareersContent/index.js

```diff
@@ -38,6 +39,29 @@ var BENEFITS = [
 ];
 
 class CareersContent extends React.Component {
+  constructor() {
+      super();
+      this.state = { activePersonIndex: -1 };
```

Comment:
Yeah, this logic should all be contained within an `ImagePopoverGrid`...well maybe just a `PopoverGrid`. I’d love to see a proposed API for what an `PopoverGrid` would look like.